pkg/report/linux.go

```diff
@@ -931,8 +931,17 @@ var linuxOopses = []*oops{
 				noStackTrace: true,
 			},
 			{
-				title: compile("BUG: sleeping function called from invalid context (.*)"),
-				fmt:   "BUG: sleeping function called from invalid context %[1]v",
+				title: compile("BUG: sleeping function called from invalid context at (.*)"),
+				fmt:   "BUG: sleeping function called from invalid context in %[2]v",
+				stack: &stackFmt{
+					parts: []*regexp.Regexp{
+						compile("Call Trace:"),
+						parseStackTrace,
+					},
+					parts2: []*regexp.Regexp{
+						compile("BUG: sleeping function called from invalid context at (.*)"),
```

Comment:
It's better to not include this additional pattern. It will lead to duplicate reports on corrupted/truncated output. It's better to mark pkg/report/testdata/linux/report/56 as corrupted because it's in fact corrupted, if that's all we've got there is not much useful info.

Reply:
Done.